edsdme/blocks/announcements-preview/announcements-preview.css

```diff
@@ -0,0 +1,75 @@
+/* stylelint-disable */
+.announcement-item,
+.link-wrapper {
+  display: flex;
+  align-items: center;
+  margin: 26px 0px;
+  background-color: white;
+  width: 100%;
+  height: 130px;
+  max-width: 600px;
+}
+
+announcements-preview .con-button {
+  margin: 5px 0px;
+}
+
+.card-image {
+  min-width: 100px;
+  width: 20%;
+  height: 100%;
+  margin-right: 20px;
+  overflow: hidden;
+}
+
+.card-image picture,
+.card-image img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.card-content {
+  height: 100%;
+  width: 80%;
+  overflow: hidden;
+  margin-right: 10px;
```

Comment:
@cod17828 shouldn't this be the same size as the gap between the image and content? currently 20px

Perhaps it's better to move the margin from the image here:
`margin: 0 20px;`